### PR TITLE
Updated some Container components to use hooks

### PR DESCRIPTION
**Pull request by wongband**
Converted the following:
- SpacesContainer
- EquipmentContainer
- PeopleContainer

To dos:
- Fix details test for each container

### CODE REVIEW

**Review by jSylvestre — CHANGES_REQUESTED** [review]
From person page.
Add a new equipment. Either create a new one or pick an existing.
Go.

Page is blank:
image

**Comment by wongband** [comment]
@jSylvestre fixed the bug in this push ^^

**Review by jSylvestre — CHANGES_REQUESTED** [review]
From within a person, try to assign an existing access fails.
Creating and assigning a new access seems to work fine.

image

File: Keas.Mvc/ClientApp/components/People/PeopleContainer.tsx [hunk 1]
```diff
@@ -1,203 +1,153 @@
...
 import CreatePerson from './CreatePerson';
 import PeopleTable from './PeopleTable';
 import PersonDetails from './PersonDetails';
 
-interface IState {
-  loading: boolean;
-  people: IPersonInfo[];
-  tableFilters: any[]; // object containing filters on table
-  tagFilters: string[]; // string of tag filters
+interface IParams {
```
Comment (srkirkland) [inline Keas.Mvc/ClientApp/components/People/PeopleContainer.tsx:15]: You don't need this, use IMatchParams

File: Keas.Mvc/ClientApp/components/Spaces/SpacesContainer.tsx [hunk 2]
```diff
@@ -15,338 +20,286 @@ import SpacesDetails from './SpacesDetails';
...
   selectedKeyInfo?: IKeyInfo;
   spacesTotalUpdated?: (keyId: number, count: number) => void;
 }
 
-interface IState {
-  spaces: ISpaceInfo[];
-  loading: boolean;
-  tableFilters: any[]; // object containing filters on table
-  tagFilters: string[]; // array of filters from SearchTags
+interface IParams {
```
Comment (srkirkland) [inline Keas.Mvc/ClientApp/components/Spaces/SpacesContainer.tsx:28]: delete and use imatchparams

**Review by srkirkland — CHANGES_REQUESTED** [review]

**Review by srkirkland — APPROVED** [review]

**Review by jSylvestre — APPROVED** [review]

**Pull request merged**